OTGTool/src/main/scala/t/global/KCDBRegistry.scala

```diff
@@ -25,37 +25,38 @@ import scala.collection.mutable.Map
 
 /**
  * The KCDBRegistry tracks open Kyoto Cabinet databases.
- * 
+ *
+ * Users can request readers and writers. Both should be closed after use.
```

Comment:
I updated the comments in this file since I was thinking through what will happen if e.g. tomcat and scalatra both have write access to the kyoto cabinet databases, and I believe file locking will ensure nothing bad happens